packages/select/src/ResourceSelect.js

```diff
@@ -101,6 +126,7 @@ const ResourceSelect = ({
     } else {
       page = 1;
     }
+
```

Comment:
I would add that same `if/else` logic for spreading the parameters here so that we have already appended `limit` and `offset` to the params. If the developer has to change the way the params look with limit or offset they can then do it with those variables as arguments to the function.

Ex.

```
parameters={({ limit, offset}) => {{
   page: (offset/limit)+1
})}
```